docker/Dockerfile

```
@@ -76,15 +76,18 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry,id=rustume-registry-${TA
 
 # =============================================================================
 # Stage 4: Web builder (WASM bindings + Vite static bundle)
+# Pinned to amd64: wasm-pack/wasm-bindgen produce arch-independent output and
+# the toolchain is more reliable on x86_64 (avoids arm64 wasm-bindgen crashes).
 # =============================================================================
-FROM chef AS web-builder
+FROM --platform=linux/amd64 rust:1.95-alpine@sha256:606fd313a0f49743ee2a7bd49a0914bab7deedb12791f3a846a34a4711db7ed2 AS web-builder
```

Hadolint (2.14.0)

[warning] Do not use --platform flag with FROM

(DL3029)